AdobeStockImageAdminUi/view/adminhtml/web/js/mediaGallery/grid/columns/image/licenseActions.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'jquery',
+    'Magento_MediaGalleryUi/js/grid/columns/image/actions',
+    'Magento_Ui/js/modal/alert'
+], function ($, Action, uiAlert) {
+    'use strict';
+
+    return Action.extend({
+        defaults: {
+            template: 'Magento_AdobeStockImageAdminUi/mediaGallery/grid/columns/image/licenseActions',
+            licenseAction: {
+                name: 'license',
+                title: 'License',
+                handler: 'licenseImageAction'
+            },
+            modules: {
+                image: '${ $.imageComponent }'
+            }
+        },
+
+        /**
+         * Initialize the component
+         *
+         * @returns {Object}
+         */
+        initialize: function () {
+            this._super().observe(['visible']);
+            this.actionsList.push(this.licenseAction);
+
+            return this;
+        },
+
+        /**
+         * License image
+         *
+         * @param {Object} record
+         */
+        licenseImageAction: function (record) {
+            this.getImageRecord(record.id);
+        },
+
+        /**
+         * Check if image licensed
+         *
+         * @param {Object} record
+         * @param {Object} name
+         */
+        isVisible: function (record, name) {
+
+            if (name === this.licenseAction.name) {
+                return parseInt(record.licensed, 16) === 0  ? true  : false;
```

Comment:
you can just return "parseInt(record.licensed, 16) === 0" it will be boolean automatically